src/db/sqlite/lrsql/sqlite/record.clj

```diff
@@ -61,7 +62,16 @@
     (create-credential-table! tx)
     (create-credential-to-scope-table! tx))
   (-update-all! [_ tx]
-    (alter-admin-account-passhash-optional! tx))
+    (log/info "Updating DB schema...")
+    (log/infof "schema version %s" (query-schema-version tx))
+    (when (= 1 (:notnull (query-admin-account-passhash-notnull tx)))
+      (let [{schema-version :schema_version} (query-schema-version tx)]
+        (log/info "Setting admin account passhash to optional")
+        (enable-writable-schema-snip tx)
+        (alter-admin-account-passhash-optional! tx)
```

Comment:
This attempted schema edit fails, the previous `enable-writable-schema-snip` does not appear to apply:

```
Execution error (SQLiteException) at org.sqlite.core.DB/newSQLException (DB.java:1012).
[SQLITE_ERROR] SQL error or missing database (table sqlite_master may not be modified)
```